pkg/expr/query.panel.schema.json

```diff
@@ -25,13 +25,17 @@
                   "type"
                 ],
                 "properties": {
+                  "apiVersion": {
+                    "description": "The apiserver version",
+                    "type": "string"
+                  },
```

Comment:
Yes -- these will need to be updated when we update the SDK.

Reply:
sorry, this is still not clear to me. The apiVersion is still not in the `datasourceRef`, and it's not clear if it will? So why are you modifying all the schemas to include it?

Reply:
ah, I missed that addition. Okay